src/ngraph/op/constant.hpp

```diff
@@ -44,8 +44,8 @@ namespace ngraph
                 : Node("Constant", {})
                 , m_element_type(type)
                 , m_shape(shape)
-                , m_data(ngraph::aligned_alloc(m_element_type.size(),
-                                               shape_size(m_shape) * m_element_type.size()))
+                , m_data(
+                      new runtime::AlignedBuffer(shape_size(m_shape) * m_element_type.size(), 64))
```

Comment:
Minor nit. Macro to define 64?

Reply:
Made a static constant to hold this magic number